e2e2/test/cases/nvidia/manifests/mpi-job-nccl-test-multi-node.yaml

```diff
@@ -69,37 +69,33 @@ spec:
             - 2G
             - -f
             - "2"
-            - -t
-            - "1"
-            - -g
-            - "1"
             - -c
             - "1"
             - -n
             - "100"
     Worker:
-      replicas: 2
+      replicas: {{.WorkerNodeCount}}
       template:
         spec:
           volumes:
           - name: dshm
             emptyDir:
               medium: Memory
           containers:
-          - image: TODO
+          - image: {{.NcclTestImage}}
             imagePullPolicy: Always
-            name: nccl-worker
+            name: nccl-test-worker
             volumeMounts:
             - mountPath: /dev/shm
               name: dshm
             resources:
-              limits:
-                nvidia.com/gpu: 8
+              requests:
+                nvidia.com/gpu: {{.GpuPerNode}}
                 hugepages-2Mi: 5120Mi
-                vpc.amazonaws.com/efa: 4
+                vpc.amazonaws.com/efa: {{.EfaInterfacePerNode}}
                 memory: 8000Mi
```

Comment:
I think we should have the capability to tune the memory, can we render the this as an object?

Reply:
Definitely not all, we will have some reserved for system/k8s.
I think this number should be fine now. But my question is more about whether we need add support training/inference tests in this package and we can build some reusable code.

Reply:
`client.RenderManifests()` is reusable and this function accepts an interface as a parameter, so we can pass any structure object to it.